README.md

```diff
@@ -42,11 +42,13 @@ Use these instructions to build a clio executable from source. These instruction
   # -------------------------------
   export BOOST_ROOT=/home/my_user/boost_1_75_0
   source ~/.profile
-5. git clone https://github.com/XRPLF/clio.git
-6. mkdir build
-7. cd build
-8. cmake ..
-9. cmake --build . -- -j$((`nproc`+1))
+5. cd ~
+6. git clone https://github.com/XRPLF/clio.git
+7. cd clio
+8. mkdir build
```

Comment:
Modern CMake allows a build/binary directory to be created when specified if it doesn't exist.
Steps 8,9,10,11 can just be
`cmake -B build && cmake --build build --parallel $(nproc)`